src/coreclr/vm/arm64/gmscpu.h

```diff
@@ -26,6 +26,8 @@ struct MachState {
     TADDR          _sp; // stack pointer after the function returns
     BOOL           _isValid;
 #ifdef __APPLE__
```

Comment:
We do this unconditionally for all Unix on x64: https://github.com/dotnet/runtime/blob/5e1caf14fd94b4ed85357002303c8935fedacb8b/src/coreclr/vm/amd64/gmscpu.h#L74-L77

Should it be the same?

Reply:
It should for the sake of having it equivalent on both architecture, I just didn't want to touch that code due to your plans to get rid of all the helper method frames soon. There is nothing wrong with the amd64 way, it is just useless on Linux.

Reply:
I'd be ok to make that change for x64 too though if you prefer that.

Reply:
Ok, let's hope we will get rid of it soon.